src/python/dxpy/cli/exec_io.py

```diff
@@ -139,7 +139,7 @@ def interactive_help(in_class, param_desc, prompt):
             if opt_num == 0:
                 query_project = dxpy.WORKSPACE_ID
             elif opt_num == 1:
-                query_project = dxpy.find_one_project(name="Reference Genome Files", public=True, billed_to="org-dnanexus", level="VIEW")['id']
+                query_project = dxpy.find_one_project(name="Reference Genome Files", public=True, billed_to="org-dnanexus_apps", level="VIEW")['id']
```

Comment:
Looks good,

Please update the prompt (line 133) --> to "Reference Genome Files"

Reply:
fixed